helpers/helpers.go

```diff
@@ -15,3 +19,8 @@ func ReverseRunes(s string) string {
 	}
 	return string(r)
 }
+
+func QuoteForSlack(message string) string {
+	var re = regexp.MustCompile(`(.*)`)
+	return re.ReplaceAllString(message, `> $1`)
```

Comment:
Hey @mourtisma j'ai pas réussi à trouver comment faire une expression régulière qui ne remplace pas les lignes vides par des `> `, tu as des billes ?

Genre 

```
Hello
How are you ?

I'm fina thanks !
```
-->
```
> Hello
> How are you ?

> I'm fina thanks !
```

Reply:
Try `(.+)` to avoid blank lines

Reply:
:facepalm: I had tried other regex like`^[^ ].*$)` but the `^` and `$` behaved strangely in go. Hopefully you had a more straightforward regex I should have thought of, thx !